### DISCUSSION TITLE
Change in behavior with small page width

### DISCUSSION THREAD

**Discussion by andrzejdoro** [squidfunk/mkdocs-material#5089] [Q&A] [answered]
When we reduce the browser width below 1220 px, then the left navigation menu hides. Is it possible to change this width to another (smaller) value? I would appreciate any guidance on this topic.

**Comment by squidfunk** [comment, marked as answer]
This is essentially the inverse of #3210, where you can find guidance how to achieve this.

**Reply by andrzejdoro** [reply]
Thanks, I'll reply in that discussion.
